packages/osd-monaco/src/xjson/README.md

```diff
@@ -1,6 +1,6 @@
 # README
 
-This folder contains the language definitions for XJSON used by the Monaco editor.
+This folder contains the language definitions for XJSON and language worker of JSON used by the Monaco editor.
```

Comment:
revert this - there's no json-specific files or functions in this `xjson` directory.
```suggestion
This folder contains the language definitions for XJSON used by the Monaco editor.
```